kuma/wiki/jinja2/wiki/document.html

```diff
@@ -87,6 +78,16 @@
   <meta name="description" content="{{ seo_summary }}">
   <meta name="twitter:description" content="{{ seo_summary }}">
   {% endif %}
+
+  <!-- if this document was saved, we need to clear out the cached draft in wiki.js, need to get the value from the request if it exists -->
```

Comment:
Please change to ``{# if this document was saved ... #}`` so it is a template-only comment.